packages/doc-site/src/components/chart-demo/LiveDemo.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { LineChart, StatusTimeline } from '@synchro-charts/react';
-import {COMPARISON_OPERATOR, DataType, StreamType} from '@synchro-charts/core';
+import { LineChart, StatusTimeline, Dial } from '@synchro-charts/react';
+import {COMPARISON_OPERATOR, DataType, StreamType, StatusIcon } from '@synchro-charts/core';
```

Comment:
looks like prettier was never ran?

Reply:
make sure to run `yarn run fix` from the root of the workspace

Reply:
Done